.github/workflows/ci.yml

```diff
@@ -167,6 +167,14 @@ jobs:
         run: |
           cargo fmt --all -- --check
 
+      - name: Tee-worker install npm deps
+        working-directory: ./tee-worker/ts-tests
+        run: corepack yarn install
+
+      - name: Tee-worker check ts code format
+        working-directory: ./tee-worker/ts-tests
+        run: corepack yarn check-format
+
```

Comment:
I think you can do it within one step, like:
```
run: |
  corepack yarn install
  corepack yarn check-format

Reply:
Yes, i know but decided to go with more granural aproach!